ipfsconn/ipfshttp/ipfshttp.go

```diff
@@ -604,6 +604,39 @@ func (ipfs *Connector) RepoStat(ctx context.Context) (*api.IPFSRepoStat, error)
 	return &stats, nil
 }
 
+// RepoGC performs a garbage collection sweep on the repo.
+func (ipfs *Connector) RepoGC(ctx context.Context) ([]api.IPFSRepoGC, error) {
+	ctx, span := trace.StartSpan(ctx, "ipfsconn/ipfshttp/RepoGC")
+	defer span.End()
+
+	ctx, cancel := context.WithTimeout(ctx, ipfs.config.IPFSRequestTimeout)
+	defer cancel()
+
+	var repoGCResp []api.IPFSRepoGC
+	res, err := ipfs.doPostCtx(ctx, ipfs.client, ipfs.apiURL(), "repo/gc", "", nil)
+	if err != nil {
+		logger.Error(err)
+		return repoGCResp, err
+	}
+	defer res.Body.Close()
+
+	dec := json.NewDecoder(res.Body)
+	for {
+		resp := api.IPFSRepoGC{}
+		if err := dec.Decode(&resp); err == io.EOF {
+			break
+		} else if err != nil {
+			return repoGCResp, err
+		}
+		if resp.Error != "" {
+			logger.Error("Error while repo gc: ", resp.Error)
+		}
+		repoGCResp = append(repoGCResp, resp)
```

Comment:
Can you investigate what the --quiet flag does, and if setting `stream-channels=false` has any effect on the output (actually making return an array instead of streaming).

Reply:
None of these options give an array instead of streaming. Strangely, for `quiet` true or false doesn't make a difference. It gives the same results. 
`stream-errors` is for whether I want to see errors in my output or not, but those are also all streaming responses.

Reply:
hmm ok